V/VIC5/build_tarballs.jl

```diff
@@ -1,23 +1,28 @@
 using BinaryBuilder
 
 name = "VIC5"
-version = v"0.1.1"
+version = v"0.1.2"
 
 # Collection of sources required to build
 sources = [
     GitSource("https://github.com/CUG-hydro/VIC5.c.git",
-    "339d9666ce17e9638e16193c43637e453e3b03ef"), # v0.1.1
+    "786c10b0bd930b761cb4fea63ed4d5fa57aa8924"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/VIC5.c/vic
+cd $WORKSPACE/srcdir/VIC5.c/vic/drivers/classic
 
-mkdir -p "${libdir}"
-mkdir -p "${bindir}"
+make CC=${CC} -j${nproc}
 
-target=${libdir}/libvic5_classic.${dlext} make dll CC=${CC} -j${nproc}
+mkdir -p ${prefix}/bin ${prefix}/lib
```

Comment:
If you use `install` to install the file you don't need to manually create destination directories.
```suggestion
```
Also, I don't know why you changed `"${libdir}"` with `${prefix}/lib`, since the latter is wrong.

Reply:
It seems that `libdir` not wokr on windows. Hence, I used `{prefix}/lib`.
```bash
[15:26:50]  ---> make install
  | [15:26:50] install -m 755 vic_classic.exe /workspace/destdir/bin
  | [15:26:50] install libvic5_classic.dll /workspace/destdir/lib
  | [15:26:50]  ---> ls $bindir
  | [15:26:50] vic_classic.exe
  | [15:26:50]  ---> ls $libdir
  | [15:26:50] vic_classic.exe
  | [15:26:50]  ---> install_license ${WORKSPACE}/srcdir/VIC5.c/LICENSE.txt

```
https://buildkite.com/julialang/yggdrasil/builds/8172#018d6f92-afa6-4d38-b06a-f3494441e06b/632-2308

I will try again.
Thanks for your careful review and suggestions.

Reply:
> It seems that libdir not wokr on windows. Hence, I used {prefix}/lib.

Well, that's wrong, because `${libdir}` on Windows is `${prefix}/bin`.